types/timepicker/index.d.ts

```diff
@@ -0,0 +1,162 @@
+// Type definitions for Timepicker Plugin for jQuery 1.13
+// Project: <https://github.com/jonthornton/jquery-timepicker>
+// Definitions by: doberkofler <https://github.com/doberkofler>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.0
```

Comment:
This is higher, than minimum version required by jQuery itself:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jquery/index.d.ts#L27
do you need this version for specific reason? (e.g. feature that cannot be downgraded via syntax change)